source/clientsideapp.md

```diff
@@ -43,26 +43,33 @@ Here we are, doing the traditionnal "Hello World" app that can make you want to
   "displayName": "Your App's Name",
   "version": "1.0.0",
   "description": "Your app's description",
-  "main": "index.html"
+  "icon-path": "main_icon.png",
+  "author": "Your name",
```

Comment:
I don't know.. so there is an example on https://github.com/cozy/cozysdk-client-tuto/blob/v1.0/package.json